### PR TITLE
Updating meeting info

### PR DESCRIPTION
**Pull request by cdavisafc**
Have pulled together changes proposed in PR #58 and PR #60 plus a few other cleanups.

### CODE REVIEW

File: MEETINGS.md [hunk 1]
```diff
@@ -1,19 +1,25 @@
 # GitOps Working Group Meetings
-Please find meetings of the GitOps Working Group available below. The next meeting will be listed at the top of the list with links to recordings of past meetings available below (when possible). Times listed are GMT.
+The GitOps Working group meets on the second Thursday of each month at 18:00 GMT - this meeting can be found in the [CNCF calendar](https://www.cncf.io/calendar/).<br>
```
Comment (scottrigby) [inline MEETINGS.md:2]:
```suggestion
The GitOps Working group meets on the second Thursday of each month at 18:00 GMT - this meeting can be found in the [CNCF calendar](https://www.cncf.io/calendar/).
```

File: MEETINGS.md [hunk 2]
```diff
@@ -1,19 +1,25 @@
...
+[Meeting Notes](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/)
 
 ## Upcoming Meetings
-Thursday, February 11th at 18:00 GMT
-[Video Conference Link](https://weaveworks.zoom.us/j/93181235360)<br>
-Meeting ID: 93181235360
+Thursday, March 11th at 18:00 GMT
+[Video Conference Link](https://zoom.us/my/cncfgitopswg?pwd=aE9FMjR0M1djd3huMlNUMWZXSXpyQT09)<br>
+Meeting Passcode: 77777
 <br><br><br>
```
Comment (scottrigby) [inline MEETINGS.md:9]:
```suggestion
### Thursday, March 11th at 18:00 GMT

- [Video Conference Link](https://zoom.us/my/cncfgitopswg?pwd=aE9FMjR0M1djd3huMlNUMWZXSXpyQT09)
- Meeting Passcode: `77777`
```

File: MEETINGS.md [hunk 3]
```diff
@@ -1,19 +1,25 @@
...
 
 ## Past Meetings
+Thursday, February 11th at 18:00 GMT
+[Video Recording](https://weaveworks.zoom.us/rec/share/OPy5MSMzgIkPX1-PAdhwrnGGa82Lr5BuVpQ1iSK-sbHPph68ML1JNVQXRKlP09bw.bjqDnPKEA2W9D2HG)<br>
+Passcode: fwPy8^fb<br>
+[Meeting Notes](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#heading=h.w91ueqcmi7nz)
```
Comment (scottrigby) [inline MEETINGS.md:15]:
```suggestion
### Thursday, February 11th at 18:00 GMT

- [Video Recording](https://weaveworks.zoom.us/rec/share/OPy5MSMzgIkPX1-PAdhwrnGGa82Lr5BuVpQ1iSK-sbHPph68ML1JNVQXRKlP09bw.bjqDnPKEA2W9D2HG)
- Passcode: `fwPy8^fb`
- [Meeting Notes](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#heading=h.w91ueqcmi7nz)
```

File: MEETINGS.md [hunk 4]
```diff
@@ -1,19 +1,25 @@
...
+
 Thursday, January 14th at 17:00 GMT
 [Video Recording](https://weaveworks.zoom.us/rec/share/_qrMRze16IHb2d4KIOHSa2Wxb_XzbU41-ZH6YZI5aJkUqOHM1bBQLCQhkFva5xo0.ay0LdxTVPCTlgbSd)<br>
 Passcode: EpQ26^cV<br>
-[Meeting Notes:](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/)
+[Meeting Notes](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#heading=h.f3snsmsnuelu)
```
Comment (scottrigby) [inline MEETINGS.md:20]:
```suggestion
- [Meeting Notes](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#heading=h.f3snsmsnuelu)
```

File: MEETINGS.md [hunk 5]
```diff
@@ -1,19 +1,25 @@
...
 
 ## Past Meetings
+Thursday, February 11th at 18:00 GMT
+[Video Recording](https://weaveworks.zoom.us/rec/share/OPy5MSMzgIkPX1-PAdhwrnGGa82Lr5BuVpQ1iSK-sbHPph68ML1JNVQXRKlP09bw.bjqDnPKEA2W9D2HG)<br>
+Passcode: fwPy8^fb<br>
+[Meeting Notes](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#heading=h.w91ueqcmi7nz)
+
 Thursday, January 14th at 17:00 GMT
 [Video Recording](https://weaveworks.zoom.us/rec/share/_qrMRze16IHb2d4KIOHSa2Wxb_XzbU41-ZH6YZI5aJkUqOHM1bBQLCQhkFva5xo0.ay0LdxTVPCTlgbSd)<br>
 Passcode: EpQ26^cV<br>
```
Comment (scottrigby) [inline MEETINGS.md:19]:
```suggestion
### Thursday, January 14th at 17:00 GMT

- [Video Recording](https://weaveworks.zoom.us/rec/share/_qrMRze16IHb2d4KIOHSa2Wxb_XzbU41-ZH6YZI5aJkUqOHM1bBQLCQhkFva5xo0.ay0LdxTVPCTlgbSd)<br>
- Passcode: `EpQ26^cV`
```

File: MEETINGS.md [hunk 6]
```diff
@@ -1,19 +1,25 @@
...
-[Meeting Notes:](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/)
+[Meeting Notes](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#heading=h.f3snsmsnuelu)
 
 Thursday, December 10th at 17:00 GMT
 [Video Recording](https://weaveworks.zoom.us/rec/share/jyagKQsL7irvi1pF-RHzrrwH-Sqa59wNnBK0F2H8QYBo_rxJZOXMO-5j5CwUgKiK.9cl0var5BN-z0qrI)<br>
 Passcode: %mi$&f8e<br>
```
Comment (scottrigby) [inline MEETINGS.md:24]:
```suggestion
### Thursday, December 10th at 17:00 GMT

- [Video Recording](https://weaveworks.zoom.us/rec/share/jyagKQsL7irvi1pF-RHzrrwH-Sqa59wNnBK0F2H8QYBo_rxJZOXMO-5j5CwUgKiK.9cl0var5BN-z0qrI)<br>
- Passcode: `%mi$&f8e`
```

File: MEETINGS.md [hunk 7]
```diff
@@ -1,19 +1,25 @@
...
-[Meeting Notes:](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/)
+[Meeting Notes](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#heading=h.e0ignly90e17)
```
Comment (scottrigby) [inline MEETINGS.md:25]:
```suggestion
- [Meeting Notes](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/edit#heading=h.e0ignly90e17)
```

**Review by scottrigby — COMMENTED** [review]
Changes look good! Just some formatting suggestions to make it more markdown-ish. Also headers give direct anchor links to specific weeks. WDYT?

**Comment by tonit** [comment]
+1 to make it more organized and link within the doc for specific meetings. In any way: I'd favor a quick merge because people probably look at the meetings doc and don't find all the good info currently.

**Comment by cdavisafc** [comment]
Yes, @tonit - I've asked for a quick merge. Added @scottrigby 's markup changes to the PR.

**Review by scottrigby — APPROVED** [review]
🚀

**Comment by scottrigby** [comment]
@cdavisafc if you second this I believe we can change the settings to require only 1 required signoff during this phase https://github.com/gitops-working-group/gitops-working-group/issues/49

**Pull request merged**